Projects/Chat/Sources/ChatStore.swift

```diff
@@ -23,6 +27,18 @@ final public class ChatStore: StateStore<ChatState, ChatAction> {
         switch action {
         case .checkPushNotificationStatus:
             newState.askedForPushNotificationsPermission = true
+        case let .setFailedMessage(conversationId, message):
+            var failedMessages = state.failedMessages
+            var messages = failedMessages[conversationId]
```

Comment:
Can we replace this with
`var messages = failedMessages[conversationId] ?? []`
Why?
How its written now, we would never have failed messages for the conversation since messages will be nil
Therefore append will never add new message
Therefore failedMessages[conversationId] will be set to nil allways